database/Cargo.toml

```diff
@@ -21,19 +22,24 @@ tracing-log = "0.1.2"
 tracing = "0.1.29"
 
 ring = "^0.16.11"
-base64 = "0.13.0"
 uuid = { version = "0.8.1", features = ["v4"] }
 cfg-if = "1.0.0"
 err-derive = "0.3.0"
-sqlx = { version = "=0.5.5", features = ["runtime-tokio-rustls"] }
+sqlx = { version = "=0.5.13", features = ["runtime-tokio-rustls"] }
```

Comment:
I'm kind of surprised updating to the latest sqlx version worked as last time I attempted it their proc macros started misbehaving.